RUNNING: "python -m distributed.cli.dask_scheduler --protocol=tcp
                    --scheduler-file /root/cugraph/python/cugraph-pyg/cugraph_pyg/examples/scheduler.json
                "
/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/cli/dask_scheduler.py:140: FutureWarning: dask-scheduler is deprecated and will be removed in a future release; use `dask scheduler` instead
  warnings.warn(
2023-04-04 04:25:23,233 - distributed.scheduler - INFO - -----------------------------------------------
2023-04-04 04:25:23,926 - distributed.scheduler - INFO - State start
2023-04-04 04:25:23,940 - distributed.scheduler - INFO - -----------------------------------------------
2023-04-04 04:25:23,941 - distributed.scheduler - INFO -   Scheduler at:  tcp://10.120.104.11:8786
2023-04-04 04:25:23,941 - distributed.scheduler - INFO -   dashboard at:  http://10.120.104.11:8787/status
2023-04-04 04:25:33,157 - distributed.scheduler - INFO - Register worker <WorkerState 'tcp://10.120.104.11:34957', status: init, memory: 0, processing: 0>
2023-04-04 04:25:33,160 - distributed.scheduler - INFO - Starting worker compute stream, tcp://10.120.104.11:34957
2023-04-04 04:25:33,160 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:33580
2023-04-04 04:25:33,167 - distributed.scheduler - INFO - Register worker <WorkerState 'tcp://10.120.104.11:45003', status: init, memory: 0, processing: 0>
2023-04-04 04:25:33,167 - distributed.scheduler - INFO - Starting worker compute stream, tcp://10.120.104.11:45003
2023-04-04 04:25:33,167 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:33592
2023-04-04 04:25:55,918 - distributed.scheduler - INFO - Receive client connection: Client-cae71d19-d2a0-11ed-9cfb-5cff35c1a711
2023-04-04 04:25:55,918 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:44668
2023-04-04 04:25:56,158 - distributed.worker - INFO - Run out-of-band function '_func_set_scheduler_as_nccl_root'
2023-04-04 04:25:56,738 - distributed.scheduler - INFO - Receive client connection: Client-caee55a7-d2a0-11ed-9cfc-5cff35c1a711
2023-04-04 04:25:56,739 - distributed.core - INFO - Starting established connection to tcp://10.120.104.11:44688
2023-04-04 04:25:57,388 - distributed.worker - INFO - Run out-of-band function '_func_set_scheduler_as_nccl_root'
2023-04-04 04:26:22,993 - distributed.scheduler - INFO - Remove client Client-cae71d19-d2a0-11ed-9cfb-5cff35c1a711
2023-04-04 04:26:22,994 - distributed.core - INFO - Received 'close-stream' from tcp://10.120.104.11:44668; closing.
2023-04-04 04:26:22,994 - distributed.scheduler - INFO - Remove client Client-cae71d19-d2a0-11ed-9cfb-5cff35c1a711
2023-04-04 04:26:22,994 - distributed.scheduler - INFO - Close client connection: Client-cae71d19-d2a0-11ed-9cfb-5cff35c1a711
2023-04-04 04:26:23,086 - distributed.scheduler - INFO - Remove client Client-caee55a7-d2a0-11ed-9cfc-5cff35c1a711
2023-04-04 04:26:23,086 - distributed.core - INFO - Received 'close-stream' from tcp://10.120.104.11:44688; closing.
2023-04-04 04:26:23,086 - distributed.scheduler - INFO - Remove client Client-caee55a7-d2a0-11ed-9cfc-5cff35c1a711
2023-04-04 04:26:23,086 - distributed.scheduler - INFO - Close client connection: Client-caee55a7-d2a0-11ed-9cfc-5cff35c1a711
2023-04-04 04:33:03,774 - distributed._signals - INFO - Received signal SIGINT (2)
2023-04-04 04:33:03,774 - distributed.scheduler - INFO - Scheduler closing...
2023-04-04 04:33:03,775 - distributed.scheduler - INFO - Scheduler closing all comms
2023-04-04 04:33:03,775 - distributed.core - INFO - Connection to tcp://10.120.104.11:33592 has been closed.
2023-04-04 04:33:03,776 - distributed.scheduler - INFO - Remove worker <WorkerState 'tcp://10.120.104.11:45003', status: running, memory: 0, processing: 0>
2023-04-04 04:33:03,776 - distributed.core - INFO - Removing comms to tcp://10.120.104.11:45003
2023-04-04 04:33:03,776 - distributed.core - INFO - Connection to tcp://10.120.104.11:33580 has been closed.
2023-04-04 04:33:03,776 - distributed.scheduler - INFO - Remove worker <WorkerState 'tcp://10.120.104.11:34957', status: running, memory: 0, processing: 0>
2023-04-04 04:33:03,776 - distributed.core - INFO - Removing comms to tcp://10.120.104.11:34957
2023-04-04 04:33:03,776 - distributed.scheduler - INFO - Lost all workers
2023-04-04 04:33:03,777 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:33580>
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 317, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 328, in write
    convert_stream_closed_error(self, e)
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 144, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:33580>: Stream is closed
2023-04-04 04:33:03,778 - distributed.batched - INFO - Batched Comm Closed <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:33592>
Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 317, in write
    raise StreamClosedError()
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/batched.py", line 115, in _background_send
    nbytes = yield coro
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/tornado/gen.py", line 769, in run
    value = future.result()
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 328, in write
    convert_stream_closed_error(self, e)
  File "/opt/conda/envs/rapids/lib/python3.10/site-packages/distributed/comm/tcp.py", line 144, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) Scheduler connection to worker local=tcp://10.120.104.11:8786 remote=tcp://10.120.104.11:33592>: Stream is closed
2023-04-04 04:33:03,779 - distributed.scheduler - INFO - Stopped scheduler at 'tcp://10.120.104.11:8786'
2023-04-04 04:33:03,779 - distributed.scheduler - INFO - End scheduler
